universal-application-tool-0.0.1/conf/messages

```diff
@@ -21,7 +21,7 @@ button.nextBlock=Save and continue
 button.deleteEntity=Delete {0}
 
 # A toast message that displays when a program is not fully localized to the applicant's preferred locale.
-toast.localeNotSupported=We're sorry, this program is not fully translated to your preferred language.
+toast.localeNotSupported=We''re sorry, this program is not fully translated to your preferred language.
```

Comment:
(blocking) please revert changes to messages. It is outside the scope of this PR, and if we need them they can be submitted in a different PR.